include/uv.h

```diff
@@ -1313,6 +1313,9 @@ enum {
 UV_EXTERN int uv_thread_getpriority(uv_thread_t tid, int* priority);
 UV_EXTERN int uv_thread_setpriority(uv_thread_t tid, int priority);
 
+UV_EXTERN int uv_set_threadpool_size(unsigned int number);
+UV_EXTERN int uv_get_threadpool_size(unsigned int* number);
```

Comment:
Why not simply return the value, rather than getting an out argument?

Reply:
Passing an arg pointer and fill its value is the most common pattern (IMO) in libuv, all calls seems to return errno like instead of values (that’s what I normally see). I can modify it for your suggestion, but I personally think that breaks the most common pattern here

Reply:
```
UV_EXTERN size_t uv_loop_size(void);
UV_EXTERN size_t uv_handle_size(uv_handle_type type);
...

UV_EXTERN int uv_send_buffer_size(uv_handle_t* handle, int* value);
UV_EXTERN int uv_recv_buffer_size(uv_handle_t* handle, int* value);

UV_EXTERN size_t uv_stream_get_write_queue_size(const uv_stream_t* stream);
UV_EXTERN size_t uv_udp_get_send_queue_size(const uv_udp_t* handle);
UV_EXTERN int uv_cpumask_size(void);

```

For size shenanigans, looks like just returning the size is the more used pattern.

Reply:
Yeap, will update that one.